vms/platformvm/txs/executor/standard_tx_executor.go

```diff
@@ -135,7 +138,7 @@ func (e *StandardTxExecutor) ImportTx(tx *txs.ImportTx) error {
 		utxoIDs[i] = utxoID[:]
 	}
 
-	if e.Bootstrapped.Get() {
+	if e.Bootstrapped.Get() && !e.Config.LiteSyncPrimaryNetwork {
```

Comment:
nit: add a comment explaining what we are skipping (import verification)

Reply:
done